index.bs

```diff
@@ -8072,8 +8072,8 @@ ECMAScript Array values.
         1.  Let |V| be the value in |S| at index |i|.
         1.  Let |E| be the result of [=converted to an ECMAScript value|converting=]
             |V| to an ECMAScript value.
-        1.  Let |P| be the result of calling <a abstract-op>ToString</a>(|i|).
-        1.  Call <a abstract-op>CreateDataProperty</a>(|A|, |P|, |E|).
+        1.  Let |P| be the result of calling [=?=] <a abstract-op>ToString</a>(|i|).
```

Comment:
This can't throw, since `i` is primitive here. (Just gonna ignore the issues with real-vs-Number for the moment.)

Reply:
Fixed.